From 13e62011600ace59c81549cbfb836ce8841fb2d2 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 10 Nov 2011 18:16:26 +0000
Subject: [PATCH 0097/1000] replacement for trace.dll, trchdl.dll and
 trcserv.exe: process writes traces to file, using same interface as trace.dll
 and trchdl.dll needs no trcserv.exe trace.dll trcHDL.dll traceNoConfig.dll
 trcHDLNoConfig.dll traces.lib tracehdls.lib traceNoConfigs.lib
 tracehdlNoConfigs.lib

git-svn-id: svn://localhost/SelfServiceCommon/trunk@215 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/Trace/trace.makefile.common.mak | 1 -
 1 file changed, 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/cpp/Trace/trace.makefile.common.mak b/SelfServiceCommon/Massai/cpp/Trace/trace.makefile.common.mak
index 763ba23a..3241909a 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/trace.makefile.common.mak
+++ b/SelfServiceCommon/Massai/cpp/Trace/trace.makefile.common.mak
@@ -5,7 +5,6 @@ MY_CFLAGS =            -GR -I$(P_INC) $(INC_INTERFACES) -DBOOST_ALL_NO_LIB $(INC
 MY_RCFLAGS =           -I$(P_INC)
 
 DLL_LDFLAGS = \
-    /LINK50COMPAT \
     /EXPORT:closeTrace \
     /EXPORT:dumpTrace \
     /EXPORT:dumpTraceForTopic \
-- 
2.41.0.windows.1

